src/java.base/share/classes/sun/security/util/ECParameters.java

```diff
@@ -123,7 +123,8 @@ protected void engineInit(AlgorithmParameterSpec paramSpec)
             namedCurve = CurveDB.lookup(keySize);
         } else {
             throw new InvalidParameterSpecException
-                ("Only ECParameterSpec and ECGenParameterSpec supported");
+                ("Only ECParameterSpec, ECGenParameterSpec " +
+                        "and ECKeySizeParameterSpec supported");
```

Comment:
Would you mind reducing the leading whitespaces of the 2nd line so that the two lines are aligned?  It would be nice to move the '9' to the previous line, as you are already there.